### PR TITLE
Add FAQ to the Anti-Xray guide

### PR DESCRIPTION
**Pull request by stonar96**

### CODE REVIEW

File: docs/paper/admin/how-to/anti-xray.md [hunk 1]
```diff
@@ -280,3 +280,45 @@ anticheat:
...
+</details>
+
+<details>
+  <summary>It still doesn't work, further troubleshooting</summary>
+
+* Make sure to always restart your server after making changes to the Anti-Xray configuration.
+  Changes won't be applied automatically.
+* Do not use the `/reload` command. To apply Anti-Xray configuration changes a restart is required.
+* After restarting the server, verify that the configuration is applied correctly by inspecting the
+  config sections with Aikar's Minecraft Timings Viewer or spark.
```
Comment (MiniDigger) [inline docs/paper/admin/how-to/anti-xray.md:310]: do we call it `Aikar's Minecraft Timings Viewer` elsewhere? that seems like a mouthfully, I would say with timings or spark, or with the timings command or via the timings viewer.

File: docs/paper/admin/how-to/anti-xray.md [hunk 2]
```diff
@@ -280,3 +280,45 @@ anticheat:
...
+
+</details>
+
+<details>
+  <summary>How and where do I ask for support if it still doesn't work?</summary>
+
+If the above bullet points don't solve your problem or if you have further questions about
+Anti-Xray, please don't hesitate to ask us on the [PaperMC Discord](https://discord.gg/papermc).
```
Comment (MiniDigger) [inline docs/paper/admin/how-to/anti-xray.md:318]: maybe insert that they should be using #paper-help here?

**Review by MiniDigger — APPROVED** [review]
sounds good, just two nitpicks

**Comment by stonar96** [comment]
Thanks @MiniDigger for the quick review. I've made the requested changes and I've added a further FAQ.

**Review by lynxplay — APPROVED** [review]

**Pull request merged**